src/ui/components/Button/index.js

```diff
@@ -9,31 +9,32 @@ import './Button.scss';
 
 export default class Button extends React.Component {
   static propTypes = {
-    appearance: PropTypes.string,
```

Comment:
Yeah, the `appearance` version isn't quite as clean as the boolean props but it makes it a lot easier to implement within the component.

There's probably something you could do to avoid writing a bunch of `classNames({ 'Button--light': this.props.light, 'Button--dark': this.props.dark, 'Button--enough-already': ... })` though.